### PR TITLE
ARTEMIS-587 add more JSON JMX mgmt methods to core

### PR DESCRIPTION
**Pull request by jbertram**

### CODE REVIEW

File: artemis-server/src/main/java/org/apache/activemq/artemis/core/management/impl/ActiveMQServerControlImpl.java [hunk 1]
```diff
@@ -1360,6 +1359,130 @@ public String listProducersInfoAsJSON() throws Exception {
...
+      checkStarted();
+
+      clearIO();
+
+      try {
+         JSONArray array = new JSONArray();
+
+         Set<RemotingConnection> connections = server.getRemotingService().getConnections();
+
+         for (RemotingConnection connection : connections) {
```
Comment (johnament) [inline artemis-server/src/main/java/org/apache/activemq/artemis/core/management/impl/ActiveMQServerControlImpl.java]: Can this wait until the json change?

Comment (jbertram) [inline artemis-server/src/main/java/org/apache/activemq/artemis/core/management/impl/ActiveMQServerControlImpl.java]: That's fine with me.

**Comment by jbertram** [comment]
I rebased this on John's recent JSON changes.

File: artemis-jms-server/src/main/java/org/apache/activemq/artemis/jms/management/impl/JMSServerControlImpl.java [hunk 2]
```diff
@@ -796,20 +796,12 @@ public String listSessionsAsJSON(final String connectionID) throws Exception {
...
-                  .add("sessionID", sess.getName())
-                  .add("creationTime", sess.getCreationTime());
-            array.add(obj);
-         }
+         return server.listSessionsAsJSON(connectionID);
       }
       finally {
          blockOnIO();
       }
-      return array.toString();
```
Comment (johnament) [inline artemis-jms-server/src/main/java/org/apache/activemq/artemis/jms/management/impl/JMSServerControlImpl.java]: Shoot, just noticed this error.  As  soon as this PR is in I'll fix the other place.

**Pull request closed without merging**